tc39-well-known-symbol-clone.md

```diff
@@ -19,22 +19,25 @@ function Clone(O) {
 
 Unfortunately, this strategy is problematic in the following cases:
 
-1. Cloning host objects
-  Despite creating the clone object using the host object's prototype, a native C++ instance of the host object is not created.
-  In some engines, accessing any properties of this object specified in IDL will result in an exception.
-2. Cloning ES6 collections
-  Data in collections is not stored as properties of the collection, and as such, copying properties does not accomplish the
-  requested task. It takes a fair bit of special casing code in order to handle cloning these objects.
+1. **Cloning host objects**
+
+	Despite creating the clone object using the host object's prototype, a native C++ instance of the host object is not created.
+
+	In some engines, accessing any properties of this object specified in IDL will result in an exception.
+
+2. **Cloning ES6 collections**
```

Comment:
I think it would be more current to refer to these collections differently now. Maybe something like "Collections introduced in ECMAScript 2015, including Map, Set, WeakMap, and WeakSet"?

Could be a separate editorial commit

Reply:
I&nbsp;only&nbsp;changed the&nbsp;formatting, as&nbsp;for&nbsp;weak&nbsp;collections, those&nbsp;don't&nbsp;actually&nbsp;have any&nbsp;way&nbsp;to&nbsp;iterate over&nbsp;their&nbsp;contents in&nbsp;implementations, which&nbsp;is&nbsp;why the&nbsp;`WeakMap.prototype.clear(…)` and&nbsp;`WeakSet.prototype.clear(…)`&nbsp;methods were&nbsp;dropped from&nbsp;the&nbsp;spec.